banyand/liaison/grpc/grpc.go

```diff
@@ -19,31 +19,54 @@ package grpc
 
 import (
 	"context"
+	"flag"
+	"fmt"
+	"io"
+	"log"
 	"net"
+	"strings"
+	"time"
 
-	"google.golang.org/grpc"
+	"github.com/pkg/errors"
+	grpclib "google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
 
+	"github.com/apache/skywalking-banyandb/api/common"
+	"github.com/apache/skywalking-banyandb/api/data"
 	"github.com/apache/skywalking-banyandb/api/event"
 	v1 "github.com/apache/skywalking-banyandb/api/proto/banyandb/v1"
+	apischema "github.com/apache/skywalking-banyandb/api/schema"
 	"github.com/apache/skywalking-banyandb/banyand/discovery"
+	serverData "github.com/apache/skywalking-banyandb/banyand/liaison/data"
 	"github.com/apache/skywalking-banyandb/banyand/queue"
 	"github.com/apache/skywalking-banyandb/pkg/bus"
+	"github.com/apache/skywalking-banyandb/pkg/convert"
 	"github.com/apache/skywalking-banyandb/pkg/logger"
+	"github.com/apache/skywalking-banyandb/pkg/partition"
+	"github.com/apache/skywalking-banyandb/pkg/query/logical"
 	"github.com/apache/skywalking-banyandb/pkg/run"
 )
 
+var (
+	tls      = flag.Bool("tls", false, "Connection uses TLS if true, else plain TCP")
```

Comment:
As I understand, the flags should be defined in the `func (s *Server) FlagSet() *run.FlagSet`? @hanahmily